paddle/fluid/distributed/ps/service/brpc_ps_client.cc

```diff
@@ -136,7 +136,8 @@ int32_t BrpcPsClient::CreateClient2ClientConnection(
     server_ip_port.append(":");
     server_ip_port.append(std::to_string(client_list[i].port));
     _client_channels[i].reset(new brpc::Channel());
-    if (_client_channels[i]->Init(server_ip_port.c_str(), "", &options) != 0) {
+    if (static_cast<int>(_client_channels[i]->Init(server_ip_port.c_str(), "",
+                                                   &options)) != 0) {
```

Comment:
```suggestion
    if (_client_channels[i]->Init(server_ip_port.c_str(), "",
                                                   &options)) {
```
Init() returns bool, do not need to compare with 0

Reply:
ok